### DISCUSSION TITLE
[自动深色模式] 根据 系统状态 切换黑夜白天模式

### DISCUSSION THREAD

**Discussion by bbb3n** [tangly1024/NotionNext#2095] [Ideas]
**为什么提出这个新的特性改动**
大部分网站都是会根据手机或者浏览器的黑暗白天模式
来切换网页的黑暗白天模式

**描述一下你推荐的解决方案**
可以保留固定时间切换黑夜白天的同时
添加一个选项，可以根据系统黑暗还是白天来切换网页

**描述一下你考虑过的其它替代解决方案**
无

**补充说明**
无

**Comment by tangly1024** [comment]
原本就支持的：

https://github.com/tangly1024/NotionNext/blob/d1268ea1bff68ad9ad6f8ad24900640a36358e8d/blog.config.js#L12-L13

https://github.com/tangly1024/NotionNext/blob/d1268ea1bff68ad9ad6f8ad24900640a36358e8d/themes/theme.js#L92-L97

**Comment by bbb3n** [comment]
> 原本就支持的：
> 
> https://github.com/tangly1024/NotionNext/blob/d1268ea1bff68ad9ad6f8ad24900640a36358e8d/blog.config.js#L12-L13
> 
> https://github.com/tangly1024/NotionNext/blob/d1268ea1bff68ad9ad6f8ad24900640a36358e8d/themes/theme.js#L92-L97

您的意思是APPEARANCE_DARK_TIME需要设置为false吗

**Comment by bbb3n** [comment]
> 原本就支持的：
> 
> https://github.com/tangly1024/NotionNext/blob/d1268ea1bff68ad9ad6f8ad24900640a36358e8d/blog.config.js#L12-L13
> 
> https://github.com/tangly1024/NotionNext/blob/d1268ea1bff68ad9ad6f8ad24900640a36358e8d/themes/theme.js#L92-L97

测试了确实可以根据浏览器状态切换
但是还有点小问题
不知道是不是作者您预料之内的
我在手机上测试中发现
原生自带浏览器（三星浏览器）可以完美跟随系统
但是edge和Chrome不是
第一次进入网站时，网站确实会根据系统状态转为白天或者黑夜
但是如果这时我手动切换系统为相反的状态
我的三星浏览器上打开的网站可以正常切换
edge和Chrome却无法切换，无论刷新多少次网页
只有清除cookies后，刷新才行（那就和首次进一样了）

作为对照，我拿github官网测试，
所有浏览器在手机系统切换模式时，
网页均能够跟随切换

所以我个人认为可能是您的实现方式
与其他网站有区别吗？

**Comment by bbb3n** [comment]
拿了两个手机，测试了7.8个浏览器，就是edge，Chrome，firefox这三家不能随意切换，其他国产浏览器都可以😂
